spec/aws-record/record/dirty_tracking_spec.rb

```diff
@@ -236,6 +260,39 @@
 
   end
 
+  describe "#update" do
+
+    before(:each) do 
+      klass.configure_client(client: stub_client)
+    end
+
+    it 'should perform a hash based attribute assignment without persisting changes' do
+      item = klass.new
+      item.mykey = SecureRandom.uuid
+      item.body = SecureRandom.uuid
+      item.save
+
+      new_key = SecureRandom.uuid
```

Comment:
I'd strongly encourage that we use explicit values in these tests. Why do we need randomness?